src/main/java/org/openrewrite/java/testing/cleanup/AssertFalseEqualToAssertNotEquals.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2022 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.testing.cleanup;
+
+import org.openrewrite.ExecutionContext;
+import org.openrewrite.Recipe;
+import org.openrewrite.TreeVisitor;
+import org.openrewrite.java.*;
+import org.openrewrite.java.search.UsesMethod;
+import org.openrewrite.java.tree.Expression;
+import org.openrewrite.java.tree.J;
+
+import java.util.List;
+import java.util.function.Supplier;
+
+public class AssertFalseEqualToAssertNotEquals extends Recipe {
+    private static final MethodMatcher ASSERT_FALSE = new MethodMatcher(
+            "org.junit.jupiter.api.Assertions assertFalse(..)");
+
+    @Override
+    public String getDisplayName() {
+        return "`assertFalse(a.equals(b))` to `assertNotEquals(a,b)`";
+    }
```

Comment:
I would probably write this as "Replace JUnit `assertFalse(a.equals(b))` with `assertNotEquals(a, b)`"

Per our [recipe naming guidelines](https://github.com/openrewrite/rewrite/blob/main/doc/adr/0002-recipe-naming.md)

Reply:
Thank you for the feedback. I applied it with b0e1cb0aae9d7973d3f2461e08d74f724f550173